tools/build-cran-package

```diff
@@ -1,34 +0,0 @@
-#!/bin/bash
```

Comment:
This file has been changed to `test-cran-package`, and now uses RHub to run CRAN checks.